### ISSUE TITLE
limel-list: action menu on list item is not accessible by keyboard

### ISSUE THREAD

**Issue by adrianschmidt** [Lundalogik/lime-elements#1215]
## Current behavior

When there is a list item with an action menu, that menu can not be accessed by keyboard navigation.

1. Go to the example _List with action menu_ in the [docs](https://lundalogik.github.io/lime-elements/versions/latest/#/component/limel-list/).
2. Set focus somewhere nearby, for example by clicking on a list item.
3. You can tab back and forth between lists, and with a list item focused, you can use arrow up and arrow down between items, but I can find no way to focus or open the menu using the keyboard.

## Expected behavior

It should be possible to navigate to, and interact with, the action menu using keyboard only.

## Environment

- **lime-elements** version: Found in 31.1.2, but probably exists in all versions since the list item action menu was introduced.
- Framework used: Reproducible in the docs.
- Logs: N/A
